crates/biome_cli/tests/commands/format.rs

```diff
@@ -1141,6 +1142,37 @@ fn format_with_configuration() {
     ));
 }
 
+#[test]
+fn format_with_js_configuration() {
+    let mut console = BufferConsole::default();
+    let mut fs = MemoryFileSystem::default();
+    fs.set_working_directory("/");
```

Comment:
Boa's default module loader didn't like it if there's no working directory, so I had to add one to the memory filesystem. Note the main module is currently loaded from a string that is read from disk by our config finder, so I expect this might need a little bit more before loading external modules will work through the memory file system.